### ISSUE TITLE
Intern cells overlapping for busbar sections doubly connected 

### ISSUE THREAD

**Issue by flo-dup** [powsybl/powsybl-diagram#332]
* **Do you want to request a *feature* or report a *bug*?**
Bug

* **What is the current behavior?**
Intern cell 

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**
Create a network with 2 busbar sections, doubly connected to each other with a breaker. The switch drawn are overlapping

* **What is the expected behavior?**
One expects that the two connections do not overlap 

* **What is the motivation / use case for changing the behavior?**
Real case of two busbar sections connections overlapping:
![screenshot](https://user-images.githubusercontent.com/66690739/148364109-f143c019-1cb9-42a6-b31b-ff01ed7b850c.png)
1B is doubly connected to 1C:
   - once with a disconnector - breaker - disconnector chain
   - once with a disconnector - breaker - breaker - disconnector - chain

**Comment by flo-dup** [comment]
@BenoitJeanson do you have any thoughts about how we should draw this double connection? Does this double connection make any sense by the way?

**Comment by BenoitJeanson** [comment]
I don't think this double connection makes any sense, nevertheless, if you encountered it, we should handle it.
The problem is that both are considered FLAT... we should limit the number of FlatCell between 2 given busBars to only one FLAT (there is no such test today), and then one will be FLAT and the other will be overlapping.

**Comment by flo-dup** [comment]
Ah, ok, thanks for explaining! I hadn't seen the FLAT origin of the problem

I saw something similar but a bit trickier in another file, it happens than two busbar sections are doubly connected with a single switch connected to a busbar section on one side with two disconnectors in paralllel, that makes two LegPrimaryBlock in an horizontal LegParallelBlock. I think that's something else, so I'll open another issue for that

**Comment by Hyralc** [comment]
Due to inactivity for more than 3 years, this issue is now closed. If you plan on solving the issue, please feel free to re-open it.
